Core/Src/bsp.c

```diff
@@ -477,3 +477,25 @@ void MCB_send_msg(uint32_t id) {
 }
 
 /*---------- Private Functions -----------------------------------------------*/
+
+/* LCD TFT SCREEN ILI9488 ####################################################*/
+
+/*---------- Private define -----------------.--------------------------------*/
+
+/*---------- Private macro ------------------.--------------------------------*/
+
+/*---------- Private variables --------------.--------------------------------*/
+enum LCD_TFT_Device{CS, RST, DC, LCD_TFT_Device_NUM};
+static const struct GPIO_Tuple LCD_TFT_Device_to_GPIO_Tuple_map[LCD_TFT_Device_NUM] = {
+    [CS]  = {.GPIO_Port = LCD_TFT_CS_GPIO_OUT_GPIO_Port, .GPIO_Pin = LCD_TFT_CS_GPIO_OUT_Pin},
+    [DC]  = {.GPIO_Port = LCD_TFT_DC_GPIO_OUT_GPIO_Port, .GPIO_Pin = LCD_TFT_DC_GPIO_OUT_Pin},
+    [RST] = {.GPIO_Port = LCD_TFT_RST_GPIO_OUT_GPIO_Port, .GPIO_Pin = LCD_TFT_RST_GPIO_OUT_Pin},
+};
```

Comment:
This construct here is used improperly: the array-map construct is used to make dictionaries/maps of similar things (for example `LED_MONO`). These GPIOs are related to one another because they are necessary to drive the LCD_TFT display but they have different functions hence they are not a LCD_TFT **device**. In this case I suggest you to use a struct (a handle) to group everything since it betters conveys the fact that you are making a sort of "class" for the LCD_TFT. If you will end up using the code I pushed on ili9488 this is already available with the `struct ILI9488_Handle`.